jenkinsfiles/UB18_Check.groovy

```diff
@@ -95,7 +95,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```

Comment:
TIMESTAMP is correct, but DATESTEMP should also be changed to DATESTAMP to be consistent.